### ISSUE TITLE
Sanity checks on RGI IDs

### ISSUE THREAD

**Issue by fmaussion** [OGGM/oggm#1745]
When users are making small mistakes on their IDs (such as forgetting a 0 or similar, it certainly happened to many, but not @ Morlocke), it would be good if OGGM could fail with a better error message than "file is not here". User friendliness and all...

**Comment by Morlocke** [comment]
Nope, definitely didn't cause me to nearly lose my sanity today, because I am not a complete tool. No sirree bob.
